.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG]: "
+labels: ["bug"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version are you using?
+      options:
+        - v6 (Beta)
+        - v5 (Stable)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: What platforms are you having the problem on?
+      multiple: true
+      options:
+        - iOS
+        - Android
+        - Windows
+        - visionOS
```

Comment:
```suggestion
        - VisionOS
```

Reply:
Nah, it's should be small, like in iOS 😄

Reply:
![image](https://github.com/react-native-video/react-native-video/assets/83216158/420e68ae-18a1-438c-b681-8cf4fad017bd)
From apple docs